crates/holochain_sqlite/src/store/p2p_agent_store.rs

```diff
@@ -255,6 +255,7 @@ impl AgentStore {
 
         Ok(out
             .into_iter()
+            .filter(|(dist, _)| *dist != u32::MAX) // Filter out Zero arcs
```

Comment:
I don't think these make sense to return when asking for nodes 'near' a basis. Agents with an empty arc don't have any basis hash logically close to their arc. Preventing these agents being returned here helps avoid mistakes by callers who aren't expecting 'near' to potentially mean 'very far away'.